app/src/datasets/chorus_dt/utils/resolvers.py

```diff
@@ -17,17 +17,15 @@ def __init__(self, api_name="nominatim", api_key=None):
 
         if api_name == "nominatim":
             geolocator = Nominatim(user_agent="data4good-beges")
-            self.p_raw_cache = "./data/raw/nominatim_responses.pkl"
-            self.p_cache = "./data/prepared/nominatim_responses.pkl"
+            self.p_raw_cache = "/data/raw/chorus-dt/nominatim_responses.pkl"
+            self.p_cache = "/data/prepared/nominatim_responses.pkl"
         elif api_name == "gmap":
             geolocator = GoogleV3(api_key=api_key, domain="maps.google.fr")
-            self.p_raw_cache = "./data/raw/gmap_responses.pkl"
-            self.p_cache = "./data/prepared/gmap_responses.pkl"
+            self.p_raw_cache = "/data/raw/chorus-dt/gmap_responses.pkl"
```

Comment:
Should we use Path variables here too? Similar to:
`RAW_DATA_PATH = "/data/raw/chorus-dt"
PREPARED_DATA_PATH = "/data/prepared"`
Maybe worth it having a config file with these? What do you think ?

Reply:
https://github.com/dataforgoodfr/batch7_beges/issues/10